content/blog/gatsby-blog/index.md

```diff
@@ -0,0 +1,323 @@
+
+### 1 Create a workflow
+
+Create the workflow file in your `develop` branch at `.github/workflows/deploy.yml`
+You can call it whatever you like as long as it is inside that directory.
+
+Add this to the file contents
+
+```yml
+name: Gatsby Publish
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: enriikke/gatsby-gh-pages-action@v2
+      with:
+        access-token: ${{ secrets.ACCESS_TOKEN }}
+        deploy-branch: master
+```
+
+
+You may create this via the github actions UI in your repo's github page.
+
+
+Notice we are using two actions, one is a standard action provided
+by github to download the repo's code in the
+automation runner (think of it as a virtual machine) and
+the other one [`gatsby-gh-pages`](https://github.com/enriikke/gatsby-gh-pages-action) is a third party provided one.
+
+I inspected the code of the latter and it seemed ok but I always recommend checking third party
+code before using it to avoid any potential security attacks.
+In this case is even more important because this action will have
+access to an `ACCESS_TOKEN` that we will be creating in the next step
+and that gives the script access to push to your repo.
+
+TLDR: be careful to whom you give these sort of permissions to.
+
+Please note that if you are doing this step you must have added the CNAME file to your source
+as explained in the section above
+
+
+### 2 Set up an access token
+
+You may have noticed this line in your workflow:
+
+```yml
+
+access-token: ${{ secrets.ACCESS_TOKEN }}
+```
+
+This is telling github actions to read a variable called `ACCESS_TOKEN` from your repo's secrets
+and pass it as a parameter named `access-token` to the `enriikke/gatsby-gh-pages-action@v2` action.
+
+This will give the action permissions to push to your master branch to deploy your code.
+
+Follow the following instructions to set up your token. Please note that the name
+is important so be sure to name it `ACCESS_TOKEN` or if you want a different
+name make sure you change the name you use in your workflow. This is just a variable,
+so treat it like so.
+
+
+
+- [create a token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)
+- [use it in your workflow](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets#creating-encrypted-secrets)
+
+
+
+## Closing
+
+At this point you will have a proper gatsby blog deployed to the cloud.
+
+Lets put a list of things you probably want to take care before you really
+consider this blog production ready
+
+- setup google analytics id to analyse traffic
+- setup https for free via settings in your github repo's
```

Comment:
nit: https = HTTPS
Also, I don't think "repo's" is necessarily incorrect since it can be a direct reference to the repo's settings but I also don't think idiomatically it is used at all.